OpenRA.Mods.Common/Orders/DemolitionOrderTargeter.cs

```diff
@@ -0,0 +1,36 @@
+#region Copyright & License Information
+/*
+ * Copyright 2007-2016 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+using System.Linq;
+using OpenRA.Traits;
+
+namespace OpenRA.Mods.Common.Orders
+{
+	class DemolitionOrderTargeter : UnitOrderTargeter
```

Comment:
Why do you keep on trying to break these targeters out into the main namespace (this is at least the fifth or sixth case)?  They are encapsulated in the parent class because they have no use to outside code.